test/e2e/storage/framework/volume_resource.go

```diff
@@ -177,6 +177,12 @@ func (r *VolumeResource) CleanupResource(ctx context.Context) error {
 			if errs := e2epv.PVPVCCleanup(ctx, f.ClientSet, f.Namespace.Name, r.Pv, r.Pvc); len(errs) != 0 {
 				framework.Failf("Failed to delete PVC or PV: %v", utilerrors.NewAggregate(errs))
 			}
+			if r.Pv != nil {
```

Comment:
the line 177 is to delete both PV and PVC, it seems the problem is it does not wait and verify those are deleted. Could we add the wait in PVPVCCleanup instead?

Reply:
yes, I thought about doing that but I was not sure who else is calling `PVPVCCleanup` and if they'd expect the same thing (that the PV is fully deleted from the apiserver after sending the delete request).

One small problem is the last parameter about the timeout, it's not available in `PVPVCCleanup` so I'd need to add it as another param which I wanted to avoid.